integration/default_test.go

```diff
@@ -98,4 +98,59 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			}).Should(ContainSubstring("AspNetCore.dll exists"))
 		})
 	})
+
+	context("image is built with BP_DOTNET_FRAMEWORK_VERSION set", func() {
+		var (
+			image  occam.Image
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("installs the version from $BP_DOTNET_FRAMEWORK_VERSION", func() {
```

Comment:
```suggestion
		it("considers BP_DOTNET_FRAMEWORK_VERSION as a version source", func() {
```

Reply:
@fg-j I believe this was resolved in the commit right above this ^